artichoke-backend/src/extn/core/regexp/syntax.rs

```diff
@@ -49,9 +49,7 @@ pub fn escape_into(text: &str, buf: &mut String) {
 #[must_use]
 pub fn is_meta_character(c: char) -> bool {
```

Comment:
This change introduced a rustfmt change which triggered a clippy lint due to a prior false negative.

I filed these bugs upstream:

- https://github.com/rust-lang/rust-clippy/issues/9163
- https://github.com/rust-lang/rust-clippy/issues/9164

This set of `allow` attributes should get clippy happy:

```suggestion
#[allow(clippy::match_like_matches_macro)]
#[allow(clippy::match_same_arms)]
pub fn is_meta_character(c: char) -> bool {
```

Reply:
Thanks for the detailed explanation and report upstream! The CI didn't run, so I was not aware of this issue when opening PR.